google/cloud/internal/log_impl.cc

```diff
@@ -22,9 +22,11 @@ namespace internal {
 
 void StdClogBackend::Process(LogRecord const& lr) {
   std::lock_guard<std::mutex> lk(mu_);
-  std::clog << lr << "\n";
-  if (lr.severity >= Severity::GCP_LS_WARNING) {
-    std::clog << std::flush;
+  if (lr.severity >= min_severity_) {
```

Comment:
optional: `if (lr.severity < min_severity_) return;` reduces indentation.

Reply:
I often find the single-entry/exit and indentation (within reason) clearer, and this is such an example, so I left it alone.